### PR TITLE
Draft: feat(compute): Choosing an Instance type

### PR DESCRIPTION
**Pull request by RoRoJ**
### Description
Added reference content: Choosing an Instance type

### CODE REVIEW

**Review by ofranc — APPROVED** [review]

File: compute/instances/reference-content/choosing-instance-type.mdx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+- [Cost-Optimized](TODO-LINK)
+- [Production-Optimized](TODO-LINK)
+
+## Instance types
+
+Check out the [Instances page](https://www.scaleway.com/en/virtual-instances/) of our website for full details on each Instance type. Alternatively, head over to the [Instance creation page of the Scaleway console ](https://console.scaleway.com/instance/servers/create) to see all our available Instance types within each range.
+
+## Instance pricing
+
+Instances are priced by the hour, with different Instance types having different hourly pricing. You can check out the current prices for Instances either in the [Instance creation page of the Scaleway console ](https://console.scaleway.com/instance/servers/create) or on the [dedicated pricing page of our website](https://www.scaleway.com/en/pricing/?tags=available).
```
Comment (ldecarvalho-doc) [inline compute/instances/reference-content/choosing-instance-type.mdx:65]:
```suggestion
Instances are priced by the hour, with different Instance types having different hourly pricing. You can check out the current prices for Instances either in the [Instance creation page of the Scaleway console](https://console.scaleway.com/instance/servers/create) or on the [dedicated pricing page of our website](https://www.scaleway.com/en/pricing/?tags=available).
```

File: compute/instances/concepts.mdx [hunk 2]
```diff
@@ -118,7 +118,7 @@ Creating an image allows you to make a complete backup of your Instance. One ima
 
 ## Instance
 
-An Instance is a computing unit, either virtual or physical, that provides resources to run your applications on. Currently Scaleway offers the following Instance types: [General Purpose](#general-purpose-instance), [Development](#development-instance), [GPU](#gpu-instance), [Stardust](#stardust-instance) and [Enterprise](#enterprise-instance).
+An Instance is a computing unit, either virtual or physical, that provides resources to run your applications on. Currently Scaleway offers the following Instance types: [General Purpose](#general-purpose-instance), [Development](#development-instance), [GPU](#gpu-instance), [Stardust](#stardust-instance) and [Enterprise](#enterprise-instance). For more information about the different ranges of Instance, see our [dedicated documentation](/compute/instances/reference-content/choosing-instance-type/).
```
Comment (fdasilva59) [inline compute/instances/concepts.mdx:121]: We are renaming the Enterprise, General Purpose, Development and Stardust ranges => to be updated

File: compute/instances/how-to/migrate-instances.mdx [hunk 3]
```diff
@@ -15,6 +15,8 @@ categories:
 
 The Scaleway platform makes it very easy to migrate your data from one Instance to another, or upgrade your Instance to a more powerful one if your requirements grow. In this how-to, we will upgrade an Instance by migrating from a [DEV1-S](/compute/instances/concepts/#development-instance) Instance to a [GP1-XS](/compute/instances/concepts/#general-purpose-instance) Instance. The new GP1-XS Instance will have the same [flexible IP](/compute/instances/concepts/#flexible-ip) as the original DEV1-S Instance.
```
Comment (fdasilva59) [inline compute/instances/how-to/migrate-instances.mdx:16]: Do we keep tag named by old ranged ?

File: compute/instances/concepts.mdx (hunk 2 above)
Comment (RoRoJ) [reply compute/instances/concepts.mdx:121]: Do you mean this should be modified to say we offer Learning, Cost-Optimized, Production-Optimized and GPU Instances?

File: compute/instances/how-to/migrate-instances.mdx (hunk 3 above)
Comment (RoRoJ) [reply compute/instances/how-to/migrate-instances.mdx:16]: From what I understood these names aren't changing yet?

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**